File: 012.c
/**
    012.
    Lee cinco datos de un archivo, y calcula el promedio.
    
    @author Daniel Cantarín <canta@canta.com.ar>
    @date   20160512
*/

#include <stdio.h>
#include <math.h>

int main() 
{
    FILE * datos;
    float valores[5];
    float promedio;
    
    datos = fopen("medicion.txt","r");
    
    fscanf( datos, "%f", &valores[0]);
    fscanf( datos, "%f", &valores[1]);
    fscanf( datos, "%f", &valores[2]);
    fscanf( datos, "%f", &valores[3]);
    fscanf( datos, "%f", &valores[4]);
    
    printf("Leí estos números: %2.2f, %2.2f, %2.2f, %2.2f, y %2.2f.\n", valores[0],valores[1],valores[2],valores[3],valores[4]);
    
    promedio = (valores[0] + valores[1] + valores[2] + valores[3] + valores[4]) / 5;
    
    printf("El promedio es %2.2f.\n", promedio);
    
    return fclose(datos);
}

File: 012.m
# Guia #4, Ej #1.
# @author   Daniel Cantarín <canta@canta.com.ar>
# @date     20161021

function [ten, cor] = leer_file( f )
    fid = fopen( f ,'r');
    ten = [];
    cor = [];
    i   = 0;
    
    fscanf( fid, '%s', 1 );
    fscanf( fid, '%s', 3 );
    fscanf( fid, '%s', 2 );
    
    
    while feof(fid) == 0
      i   = i + 1;
      tmp = fscanf( fid, '%f', 2);
      #tmp
      if size(tmp) > 0
        ten(i)  = tmp(1);
        cor(i)  = tmp(2);
      end
    end
    
endfunction

